Dockerfile

```diff
@@ -57,6 +54,7 @@ RUN apt-get update && \
         mysql-client \
         poppler-utils \
         sudo \
+        vim \
```

Comment:
@tiborsimko 

> or they can do something like `docker exec -i -t --user root myapp_web_1 /bin/bash` and, when inside, then `apt-get install vim` as usual.

Not possible because the images are mounted read-only by default. I vote for installing `nano` as a neutral, minimal editor. Right now I'm using `sed` hacks as a work-around and it is really difficult to do file modifications (for config changes or when inserting trace-points in 3rd-party libraries).